hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import styles from "./TeamDetail.module.scss";
+import Header from "components/general/Header/Header";
+import Typography from "@material-ui/core/Typography";
+import { mockTeamMultiple } from "testing/mockData";
+import {
+    Table,
+    TableBody,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TableCell,
+    Checkbox,
+    Paper,
+    Container,
+    Box,
+    Grid,
+} from "@material-ui/core";
+
+const TeamDetail = () => {
+    return (
+        <>
+            <Header />
+            <Container>
+                <Grid container direction="column" spacing={6}>
```

Comment:
I'm confused why you need a Grid for this, <Typography variant="h1"> should be fine on its own for the title

Reply:
Actually, I see what ur doing with the Grid and it's pretty nice 👍 , you can ignore these comments